setup.py

```diff
@@ -11,47 +11,33 @@ def read_file(filename):
         content = f.read()
     return content
 
+
 README = read_file('README.rst')
 CHANGELOG = read_file('CHANGELOG.rst')
 
-REQUIREMENTS = [
-    "pyramid>=1.9.1,<2.0",
-    "kinto[postgresql,memcached,monitoring]>=9.2.2,<10.0",
-    "kinto-attachment>=3.0,<3.1",
-    "kinto-amo>=1.0.1,<1.1.0",
-    "kinto-changes>=1.1.1,<1.2.0",
-    "kinto-elasticsearch>=0.3.0,<0.4",
-    "kinto-emailer>=1.0.2,<1.1",
-    "kinto-signer>=3.2.4,<3.3",
-    "kinto-fxa[scripts]>=2.5.0,<2.6",
-    "kinto-ldap>=0.3.0,<0.4",
-    "amo2kinto>=3.2,<3.3",
-    "boto>=2.46,<2.47",
```

Comment:
You don't. :)

The file that matters is `requirements/default.txt` and `requirements/prod.txt`.
The file `requirements/constraints.txt`  is not important. If we accidentally have a package too many in that file (for example, something we forgot to clean up) it doesn't matter much because it doesn't get installed.

The only way to know if we have a package in `requirements/default.txt` that is no longer needed requires manual insight. One way to automate that is to perhaps run coverage.py when we start uwsgi and then run the smoke tests. Then you'd need to cross-reference that back to the package names.